aii-pxelinux/src/main/perl/pxelinux.pm

```diff
@@ -512,12 +577,16 @@ foreach my $i (qw(configure boot rescue livecd firmware install)) {
     *{$name} = sub {
         my ($self, $cfg) = @_;
 
-        pxeprint($cfg) if ($i eq 'configure');
+       foreach my $variant (PXE_VARIANT_PXELINUX, PXE_VARIANT_GRUB2) {
+            if ( $VARIANT_PARAMS[$variant]->{enabled} ) {
```

Comment:
You understood properly. I didn't pay attention to the status so far, so this may need some improvement, you are right...